.chronus/changes/support-variable-base-nested-config-2024-8-25-19-8-41.md

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
might want to add another changelog entry(another file) that mark the `config-invalid-name` breaking change